CHANGELOG.md

```diff
@@ -39,6 +39,7 @@
 
 ### New
 
+- Add Cassandra Scaler ([#1841](https://github.com/kedacore/keda/pull/1841))
```

Comment:
Can you move this to the new section above please? We have released v2.3.0 last week.
```suggestion
```